coldfront/plugins/qumulo/tasks.py

```diff
@@ -97,6 +102,52 @@ def ingest_quotas_with_daily_usage() -> None:
     __validate_results(quota_usages, logger)
 
 
+def addUsersToADGroup(
+    wustlkeys: list[str], acl_allocation: Allocation, bad_keys: list = []
+) -> None:
+    if len(wustlkeys) == 0:
+        if len(bad_keys) > 0:
+            __send_invalid_users_email(acl_allocation, bad_keys)
+        return
+
+    wustlkey = wustlkeys[0]
+    group_name = acl_allocation.get_attribute("storage_acl_name")
+    active_directory_api = ActiveDirectoryAPI()
+
+    success = False
+    try:
+        active_directory_api.get_user(wustlkey)
+        success = True
+    except ValueError:
+        bad_keys.append(wustlkey)
+        success = False
+
+    if success:
+        active_directory_api.add_user_to_ad_group(wustlkey, group_name)
```

Comment:
The way this code is written, it assumes that `add_user_to_ad_group` will succeed.  That's probably generally true, but it might be worth something to log the case if/when it isn't.

Reply:
Technically, the underlying method called (`ad_add_members_to_groups`) can return False.

Reply:
Fair...I think I just got lazy here.  Will correct